package.json

```diff
@@ -25,23 +25,24 @@
     "@discipl/core-baseconnector": "0.2.2",
     "express": "4.17.1",
     "json-stable-stringify": "1.0.1",
+    "lodash": "4.17.20",
```

Comment:
With respect to dependencies, lodash is a little bit heavier than I'm comfortable pulling in. If we have a smaller stable library that provides this functionality, I'd be okay with pulling that in (like json-stable-stringify). Alternatively, we could include our own utility function.

Reply:
The Lodash.cloneDeep implementation seems pretty complex. I found a package that allowed the import of only that function.